examples/widgets/pickPositionWidget.js

```diff
@@ -0,0 +1,37 @@
+/* global itowns */
+/* global Widget */
+/* global widgetTab */
+
+const pickWidget = document.createElement('div');
+pickWidget.id = 'pickWidget';
+
+const widgetsP = document.getElementById('widgets');
+
+widgetsP.appendChild(pickWidget);
+
+const widget = new Widget(
+    'Pick',
+    document.getElementById('pickWidget'),
+    {
+        div: widgetsP,
+    });
+
+const callback = function () {
+    widgetTab.addWidget(widget);
+    if (document.getElementById('pickList')) {
+        document.getElementById(widget.pElement.id).removeChild(document.getElementById('pickList'));
+    }
+    const tab = ['longitude', 'latitude', 'altitude'];
+    const coords = itowns.viewer.pickPosition().coordinate;
+    const list = document.createElement('ul');
+    list.id = 'pickList';
+    for (var j = 0; j < coords.length; j++) {
```

Comment:
(if for..of is supported by our target browsers, we won't use babel here I guess)